tests/python/unittest/test_numpy_op.py

```diff
@@ -418,6 +418,144 @@ def hybrid_forward(self, F, a, b):
                                        rtol=1e-1, atol=1e-1, dtype=dtype)
 
 
+@with_seed()
+@use_np
+def test_np_matmul():
+    class TestMatmul(HybridBlock):
+        def __init__(self):
+            super(TestMatmul, self).__init__()
+
+        def hybrid_forward(self, F, a, b):
+            return F.np.matmul(a, b)
+
+    def matmul_backward(a, b):
+        if (a.ndim < 1) or (b.ndim < 1):
+            raise ValueError('An input is zero-dim')
```

Comment:
I think this case is never going to be hit, you can simply remove this raise from this function